sympy/stats/crv.py

```diff
@@ -153,11 +153,54 @@ def __new__(cls, dist, size):
     @classmethod
     def _sample_scipy(cls, dist, size):
         """Sample from SciPy."""
-        # scipy does not require map as it can handle using custom distributions
-        from scipy.stats import rv_continuous
+        import scipy.stats as st
```

Comment:
Can we just use the full name, `scipy.stats`? Will make code more readable as there isn't a big difference of characters between `scipy.stats` and `st`.

Reply:
I will change it on Friday. I'm not sure why I chose `st` in the first place. Maybe it was in other parts of the code.